### PR TITLE
KIK-8357-fix_layout_wallet

### PR DESCRIPTION
**Pull request by shayb**
check screen height and change to scroll layout in edge case

### CODE REVIEW

File: sample/src/main/java/kin/sdk/core/sample/Utils.java [hunk 1]
```diff
@@ -18,4 +21,25 @@ public static void copyToClipboard(Context context, CharSequence textToCopy) {
             clipboard.setPrimaryClip(clip);
         }
     }
+
+    public static int getScreenHeight(Activity activity) {
+        Display display = activity.getWindowManager().getDefaultDisplay();
+        Point size = new Point();
+        display.getSize(size);
+        int height = size.y;
+        return height;
```
Comment (berryve) [inline sample/src/main/java/kin/sdk/core/sample/Utils.java]: prefer a single getScreenSize method that returns Point instead of code duplication.

Comment (shayb) [reply sample/src/main/java/kin/sdk/core/sample/Utils.java]: that class the use it doenst need to know about the format of the answer
for now we only need the height so i can remove the the getScreen width if you prefer

Comment (berryve) [reply sample/src/main/java/kin/sdk/core/sample/Utils.java]: remove

**Comment by shayb** [comment]
all changes were done in https://github.com/kinfoundation/kin-sdk-core-android/pull/70

**Pull request closed without merging**